pkg/commands/root.go

```diff
@@ -170,5 +171,4 @@ func initRootFlagSet(fs *pflag.FlagSet, cfg *config.Config, needVersionOption bo
 	}
 
 	fs.StringVar(&cfg.Output.Color, "color", "auto", wh("Use color when printing; can be 'always', 'auto', or 'never'"))
-	fs.BoolP("help", "h", false, wh("Help for this command"))
```

Comment:
Without this, help commands does not get colored
![Screenshot from 2023-12-19 17-06-04](https://github.com/golangci/golangci-lint/assets/66965232/55a2a78f-fb04-49fa-93ad-b094fae4e8fe)

Reply:
Yes It's expected.